jobs/webcompat-kb/sql/webcompat_knowledge_base/views/site_reports/view.sql

```diff
@@ -0,0 +1,20 @@
+SELECT
+  bugs.*,
+  manager,
+  team,
+  TRIM(LOWER(IFNULL(IFNULL(JSON_VALUE(bugs.user_story, "$.diagnosis-team"), component_owners.team), "unknown"))) as assigned_team,
+  "webcompat:sitepatch-applied" IN UNNEST(bugs.keywords) AS has_intervention,
+  `{{ ref('WEBCOMPAT_BLOCKED_REASON') }}`(bugs.keywords, bugs.user_story) IS NOT NULL AS blocked,
+  `{{  ref('WEBCOMPAT_BLOCKED_REASON') }}`(bugs.keywords, bugs.user_story) AS blocked_reason,
+  --Categories for the metric
+  "webcompat:needs-diagnosis" IN UNNEST(bugs.keywords) AS metric_type_needs_diagnosis,
+  "webcompat:needs-diagnosis" NOT IN UNNEST(bugs.keywords) AND ("webcompat:platform-bug" IN UNNEST(bugs.keywords) OR
+     EXISTS (SELECT * FROM `{{ ref('breakage_reports_core_bugs') }}` where breakage_bug = bugs.number)) AS metric_type_platform_bug,
+  "blocked" = IFNULL(JSON_VALUE(bugs.user_story, "$.impact"), "") AS metric_type_firefox_not_supported,
+FROM
+  {{ ref('bugzilla_bugs') }} AS bugs
+LEFT JOIN
+  `{{ ref('bugzilla_components_ownership') }}` as component_owners ON (component_owners.bugzilla_product = product AND component_owners.bugzilla_component = component)
+WHERE
+  ((bugs.product = "Web Compatibility" AND bugs.component = "Site Reports")
```

Comment:
And this query looks different in production, mostly missing platform detection (`is_mobile` that's used in `scored_site_reports`)